### DISCUSSION TITLE
Updater returns 403

### DISCUSSION THREAD

**Discussion by fsbis** [microweber/microweber#1062] [General]
Whenever I try to update via the standalone (and any other ui way) I get a 403. Is anyone else seeing this?

**Comment by phausleitner** [comment]
yes.. https://github.com/microweber/microweber/issues/1049

**Reply by phausleitner** [reply]
Fixed, from 2.0.13 to 2.0.14 it works

**Comment by travisft** [comment]
I got this from the [Issues section](https://github.com/microweber/microweber/issues/1049#issuecomment-1984094692) of this forum, and it worked, but for me, I  had to delete a [RewriteCond %{Request_URI} /Userfile/ line write under RewriteEngine On, first. Yours may look just like the example. Mine did not. As you can see, this is not my area of expertise.
